feature-group-definitions/details.yml

```diff
@@ -1,4 +1,5 @@
 name: <details>
+description: The `<details>` element creates a disclosure widget which content is visible only when the widget is toggled open, such as when using a nested `<summary>` element.
```

Comment:
I'm not 100% sure about the "toggled open" part. How about "... which can be **expanded** to reveal ..."

Also, do you think we should mention the fact that multiple details with the same name work as a mutually exclusive accordion?

Reply:
Saying "expanded" sounds good to me. I think it's best not to mention the `name` attribute since it's a separate feature. If we had some way to link other features in descriptions we could say "see also", but we don't correctly.